deploy/helm/magda-dev.yml

```diff
@@ -115,73 +115,88 @@ minion-broken-link:
 
 connectors:
   config:
+    ## The service's image as per the global options under "global.image", but with a "name "
+    ## currently available are magda-ckan-connector, magda-project-open-data-connector and magda-csw-connector
+    - image:
+        name: "magda-ckan-connector"
+        # tag: vx.x.x
+        # pullPolicy: IfNotPresent
+        # imagePullSecret: ""
+
+      ## Unique id to identify this connector and records that are harvested from it
+      id: dga
```

Comment:
connector with ID `dga` seems defined twice.
There is another one defined around line 196